rotkehlchen/db/schema.py

```diff
@@ -696,6 +696,11 @@
 );
 """
 
+DB_CREATE_CACHE = """CREATE TABLE IF NOT EXISTS cache (
+    name TEXT NOT NULL PRIMARY KEY,
```

Comment:
I believe it had a different name previously. Can you maybe nameit `key_value_cache`

```suggestion
DB_CREATE_CACHE = """CREATE TABLE IF NOT EXISTS cache (
    name TEXT NOT NULL PRIMARY KEY,
```